benchmarks/cpp/utils/utils.py

```diff
@@ -70,13 +70,28 @@ def multimodal_dataset_dump(multimodal_texts, multimodal_image_paths,
         json.dump(workload.model_dump(), f)
 
 
-def print_text_dataset(input_ids, output_lens):
+def print_text_dataset(input_ids, output_lens, task_ids=None, lora_config=None):
     for i, input_tokens in enumerate(input_ids):
         d = {
             "task_id": i,
             "input_ids": input_tokens,
             "output_tokens": output_lens[i]
         }
+
+        # Add LoRA request if task_ids indicate LoRA usage
+        if task_ids is not None and lora_config is not None:
+            task_id = task_ids[i]
+            if task_id != -1:  # -1 means no LoRA
```

Comment:
Just to check -- you can use the random task id option without LoRA right? Wouldn't that break this logic?

Reply:
i don't think the task ids or rand task ids serve any other purpose... so i don't think it will get used for anything else. if they are used without any lora directory specified, then that's probably not intended - but thank you for the catch; currently that isn't handled properly...
I will either error out or basically send a warning and pretend it isn't a lora situation 👍🏾